### ISSUE TITLE
config/app.php is missing

### ISSUE THREAD

**Issue by HarderWork** [josegonzalez/app#2]
This is a (multiple allowed):

* [ ] bug
* [x] enhancement
* [ ] feature-discussion (RFC)

* CakePHP Application Skeleton Version: 08c9a9678dde281b08e4de1fb9ab95f2a39563cf
* Platform and Target: FriendsOfCake/vagrant-chef w. Ubuntu 16.04

### What you did
Created new project with "composer create-project --prefer-dist josegonzalez/app caketest"

### What happened
When going to the start page, I got an blank page with the message: 
Could not load configuration file: /vagrant/apps/caketest/config/app.php

### What you expected to happen
Cakes Welcome page with a notice to rename/copy app.default.php to app.php.
Or ship an app.php alongside, but have a warning like the one for changing the value of Security.salt.

**Comment by josegonzalez** [comment]
Works here:

```
jose@Joses-MacBook-Pro:~/Dropbox/src  $ composer create-project --prefer-dist josegonzalez/app caketest
Installing josegonzalez/app (1.6.0)
  - Installing josegonzalez/app (1.6.0) Downloading: 100%
Created project in caketest
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 60 installs, 0 updates, 0 removals
  - Installing cakephp/plugin-installer (1.0.0) Loading from cache
  - Installing mobiledetect/mobiledetectlib (2.8.25) Loading from cache
  - Installing psr/http-message (1.0.1) Loading from cache
  - Installing zendframework/zend-diactoros (1.4.0) Loading from cache
  - Installing psr/log (1.0.2) Loading from cache
  - Installing aura/intl (3.0.0) Loading from cache
  - Installing cakephp/chronos (1.1.2) Loading from cache
  - Installing cakephp/cakephp (3.4.9) Loading from cache
  - Installing symfony/yaml (v3.3.4) Downloading: 100%
  - Installing symfony/debug (v3.3.4) Downloading: 100%
  - Installing symfony/polyfill-mbstring (v1.4.0) Loading from cache
  - Installing symfony/console (v3.3.4) Downloading: 100%
  - Installing symfony/filesystem (v3.3.4) Downloading: 100%
  - Installing symfony/config (v3.3.4) Downloading: 100%
  - Installing robmorgan/phinx (v0.6.5) Downloading: 100%
  - Installing cakephp/migrations (1.6.7) Loading from cache
  - Installing m1/env (2.1.0) Downloading: 100%
  - Installing josegonzalez/dotenv (2.1.0) Loading from cache
  - Installing friendsofcake/bootstrap-ui (v0.6.2) Loading from cache
  - Installing friendsofcake/crud (4.4.4) Loading from cache
  - Installing friendsofcake/crud-view (0.4.3) Downloading: 100%
  - Installing friendsofcake/search (1.2.4) Downloading: 100%
  - Installing seld/cli-prompt (1.0.3) Loading from cache
  - Installing league/climate (3.2.1) Downloading: 100%
  - Installing markstory/mini-asset (1.4.1) Loading from cache
  - Installing markstory/asset_compress (3.2.3) Loading from cache
  - Installing league/flysystem (1.0.40) Loading from cache
  - Installing josegonzalez/cakephp-upload (3.7.0) Downloading: 100%
  - Installing league/event (2.1.2) Downloading: 100%
  - Installing josegonzalez/queuesadilla (0.0.7) Loading from cache
  - Installing josegonzalez/cakephp-queuesadilla (0.3.5) Downloading: 100%
  - Installing monolog/monolog (1.23.0) Loading from cache
  - Installing sentry/sentry (1.7.0) Loading from cache
  - Installing mindscape/raygun4php (v1.8.0) Loading from cache
  - Installing airbrake/phpbrake (v0.3.1) Loading from cache
  - Installing guzzlehttp/psr7 (1.4.2) Loading from cache
  - Installing guzzlehttp/promises (v1.3.1) Loading from cache
  - Installing guzzlehttp/guzzle (6.3.0) Loading from cache
  - Installing composer/ca-bundle (1.0.7) Loading from cache
  - Installing bugsnag/bugsnag (v3.5.1) Loading from cache
  - Installing josegonzalez/php-error-handlers (0.1.0) Loading from cache
  - Installing jakub-onderka/php-console-color (0.1) Downloading: 100%
  - Installing jakub-onderka/php-console-highlighter (v0.3.2) Downloading: 100%
  - Installing dnoegel/php-xdg-base-dir (0.1) Downloading: 100%
  - Installing nikic/php-parser (v3.0.6) Downloading: 100%
  - Installing symfony/var-dumper (v3.3.4) Downloading: 100%
  - Installing psy/psysh (v0.8.8) Loading from cache
  - Installing jdorn/sql-formatter (v1.2.17) Downloading: 100%
  - Installing symfony/process (v3.3.4) Downloading: 100%
  - Installing symfony/finder (v3.3.4) Loading from cache
  - Installing seld/phar-utils (1.0.1) Loading from cache
  - Installing seld/jsonlint (1.6.1) Loading from cache
  - Installing justinrainbow/json-schema (5.2.1) Loading from cache
  - Installing composer/spdx-licenses (1.1.6) Loading from cache
  - Installing composer/semver (1.4.2) Loading from cache
  - Installing composer/composer (1.4.2) Loading from cache
  - Installing cakephp/debug_kit (3.10.4) Downloading: 100%
  - Installing cakephp/bake (1.3.7) Downloading: 100%
  - Installing squizlabs/php_codesniffer (2.9.1) Loading from cache
  - Installing cakephp/cakephp-codesniffer (2.4.0) Loading from cache
josegonzalez/app suggests installing dereuromark/cakephp-ide-helper (After baking your code, this keeps your annotations in sync with the code evolving from there on for maximum IDE and PHPStan compatibility.)
josegonzalez/app suggests installing phpunit/phpunit (Allows automated tests to be run without system-wide install.)
cakephp/cakephp suggests installing lib-ICU (The intl PHP library, to use Text::transliterate() or Text::slug())
symfony/console suggests installing symfony/event-dispatcher ()
m1/env suggests installing m1/vars (For loading of configs)
friendsofcake/crud suggests installing neomerx/json-api (Framework agnostic JSON API (jsonapi.org) implementation)
markstory/mini-asset suggests installing natxet/CssMin (For using the CssMin filter.)
markstory/mini-asset suggests installing tchwork/jsqueeze (For using the JSqueeze filter.)
markstory/mini-asset suggests installing tedivm/jshrink (For using the JShrink filter.)
markstory/mini-asset suggests installing leafo/scssphp (For using the ScssPHP filter.)
markstory/mini-asset suggests installing oyejorge/less.php (For using the LessDotPHP filter, see https://github.com/oyejorge/less.php)
markstory/mini-asset suggests installing zendframework/diactoros (The middleware layer relies on zendframework/diactoros.)
markstory/asset_compress suggests installing natxet/CssMin (For using the CssMin filter.)
markstory/asset_compress suggests installing tchwork/jsqueeze (For using the JSqueeze filter.)
markstory/asset_compress suggests installing tedivm/jshrink (For using the JShrink filter.)
markstory/asset_compress suggests installing leafo/scssphp (For using the ScssPHP filter.)
markstory/asset_compress suggests installing leafo/lessphp (For using the LessPHP filter.)
league/flysystem suggests installing league/flysystem-aws-s3-v2 (Allows you to use S3 storage with AWS SDK v2)
league/flysystem suggests installing league/flysystem-aws-s3-v3 (Allows you to use S3 storage with AWS SDK v3)
league/flysystem suggests installing league/flysystem-azure (Allows you to use Windows Azure Blob storage)
league/flysystem suggests installing league/flysystem-cached-adapter (Flysystem adapter decorator for metadata caching)
league/flysystem suggests installing league/flysystem-copy (Allows you to use Copy.com storage)
league/flysystem suggests installing league/flysystem-eventable-filesystem (Allows you to use EventableFilesystem)
league/flysystem suggests installing league/flysystem-rackspace (Allows you to use Rackspace Cloud Files)
league/flysystem suggests installing league/flysystem-sftp (Allows you to use SFTP server storage via phpseclib)
league/flysystem suggests installing league/flysystem-webdav (Allows you to use WebDAV storage)
league/flysystem suggests installing league/flysystem-ziparchive (Allows you to use ZipArchive adapter)
league/flysystem suggests installing spatie/flysystem-dropbox (Allows you to use Dropbox storage)
josegonzalez/queuesadilla suggests installing pda/pheanstalk (Adds support for Beanstalk background workers)
josegonzalez/queuesadilla suggests installing iron-io/iron_mq (Adds support for IronMQ background workers)
monolog/monolog suggests installing aws/aws-sdk-php (Allow sending log messages to AWS services like DynamoDB)
monolog/monolog suggests installing doctrine/couchdb (Allow sending log messages to a CouchDB server)
monolog/monolog suggests installing ext-amqp (Allow sending log messages to an AMQP server (1.0+ required))
monolog/monolog suggests installing ext-mongo (Allow sending log messages to a MongoDB server)
monolog/monolog suggests installing graylog2/gelf-php (Allow sending log messages to a GrayLog2 server)
monolog/monolog suggests installing mongodb/mongodb (Allow sending log messages to a MongoDB server via PHP Driver)
monolog/monolog suggests installing php-amqplib/php-amqplib (Allow sending log messages to an AMQP server using php-amqplib)
monolog/monolog suggests installing php-console/php-console (Allow sending log messages to Google Chrome)
monolog/monolog suggests installing rollbar/rollbar (Allow sending log messages to Rollbar)
monolog/monolog suggests installing ruflin/elastica (Allow sending log messages to an Elastic Search server)
sentry/sentry suggests installing immobiliare/sentry-php (Fork that fixes support for PHP 5.2)
symfony/var-dumper suggests installing ext-symfony_debug ()
psy/psysh suggests installing ext-pdo-sqlite (The doc command requires SQLite to work.)
psy/psysh suggests installing hoa/console (A pure PHP readline implementation. You'll want this if your PHP install doesn't already support readline or libedit.)
cakephp/debug_kit suggests installing ext-sqlite (DebugKit needs to store panel data in a database. SQLite is simple and easy to use.)
Writing lock file
Generating autoload files
> Cake\Composer\Installer\PluginInstaller::postAutoloadDump
> App\Console\Installer::postInstall
Created `config/app.php` file
Updated __APP_NAME__ value in config/.env.default
Created `config/.env` file
Set Folder Permissions ? (Default to Y) [Y,n]? Y
Permissions set on /Users/jose/Dropbox/src/caketest/tmp/cache
Permissions set on /Users/jose/Dropbox/src/caketest/tmp/cache/models
Permissions set on /Users/jose/Dropbox/src/caketest/tmp/cache/persistent
Permissions set on /Users/jose/Dropbox/src/caketest/tmp/cache/views
Permissions set on /Users/jose/Dropbox/src/caketest/tmp/sessions
Permissions set on /Users/jose/Dropbox/src/caketest/tmp/tests
Permissions set on /Users/jose/Dropbox/src/caketest/tmp
Permissions set on /Users/jose/Dropbox/src/caketest/logs
Updated Security.salt value in config/app.php
Updated Security.salt value in config/.env.default

jose@Joses-MacBook-Pro:~/Dropbox/src  $ ll caketest/config/app.php
-rw-r--r--@ 1 jose  staff    14K Jul  6 07:44 caketest/config/app.php

jose@Joses-MacBook-Pro:~/Dropbox/src  $ php -l caketest/config/app.php
No syntax errors detected in caketest/config/app.php
```

Are you sure the `create-project` command ran successfully?

**Comment by HarderWork** [comment]
You are right, something went wrong when composer ran (complaining about missing php packages). After manually installing these on Ubuntu, I ran composer update and didn't realise there were more to come in the script.
